### DISCUSSION TITLE
NoiseModelling Days 2022

### DISCUSSION THREAD

**Discussion by pierromond** [Universite-Gustave-Eiffel/NoiseModelling#500] [Show and tell]
**If you see this message, you are invited to participate to the “NoiseModelling days 2022” on September 15th and 16th, 2022 from 10am to 4.30pm (French, UTC+2) in the form of a webinar**.

In 2020, we organized the first NoiseModelling days in the form of a general presentation of the tool and tutorials. In 2021, we focused on the users and their NM uses through scientific presentations. 

This year 2022, we will focus on practice with a first day dedicated to new noisemodelling users (beginner to advanced) and a second day for expert NM uses and code contribution.

We will still keep some user slots for you to present your work with NM. If you are interested, please send an email directly to pierre.aumond@univ-eiffel.fr

The final goal is to federate a community around NoiseModelling.

### Participate

Participation in the seminar is **free of charge** but registration is required.

Participants must register by September 13th, 2022. To do so, fill the form here: https://forms.gle/jgr9sdgKoj9RYQYa7

Practical information will be sent to registered persons.

### Program

The program is available below or here : https://noise-planet.org/noisemodelling_days.html

![NM_Days_2022_program](https://user-images.githubusercontent.com/1871248/188132944-849248fd-68fa-43e1-a777-e97ff3f944a4.png)

**Comment by Xenotech81** [comment]
Hello,
Thank you for the workshop today!

I forgot to tell, but the Docker image works now. I have tested the import of files, the calculation (Noise_level_from_traffic) and file export. There are also some instructions on how to create the container: [xenotech/noisemodelling](https://hub.docker.com/r/xenotech/noisemodelling)

It is quite big with 950MB, maybe I can reduce its size by using an Alpine Linux base image, if I find the time.



Regards
Alex
